### DISCUSSION TITLE
Opting out of Acheivement Badges

### DISCUSSION THREAD

**Discussion by askingalot** [community/community#52096] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Frankly, I don't understand the appeal of "achievements". If others like them, that's fine, but I'd like the ability to opt out.

**Comment by meppu** [comment, marked as answer]
You can disable them from your profile. Go to account settings, in "Profile settings" section disable "Show Achievements on my profile" and click "Update preferences":

![image](https://user-images.githubusercontent.com/112294217/230180505-06f502df-7c6d-42b6-979e-43b00a4adfae.png)
![image](https://user-images.githubusercontent.com/112294217/230180520-4a93719c-e820-4ea6-84f7-f399cf749bd2.png)
![image](https://user-images.githubusercontent.com/112294217/230180556-46c9ef36-689d-4369-8cad-da593d327d91.png)

**Reply by askingalot** [reply]
I completely missed that. Thank you.
